src/storage/v2/disk/storage.cpp

```diff
@@ -1771,6 +1771,9 @@ utils::BasicResult<StorageManipulationError, void> DiskStorage::DiskAccessor::Co
         case MetadataDelta::Action::POINT_INDEX_CREATE:
         case MetadataDelta::Action::POINT_INDEX_DROP:
           throw utils::NotYetImplemented("Point index is not implemented for DiskStorage.");
+        case MetadataDelta::Action::VECTOR_INDEX_CREATE:
+        case MetadataDelta::Action::VECTOR_INDEX_DROP:
+          throw utils::NotYetImplemented("Vector index is not implemented for DiskStorage.");
```

Comment:
Append "Consider switching to the IN_MEMORY_TRANSACTIONAL mode or contact the Memgraph team." @katarinasupe what's the best way of instructing users to contact us? 🤔